src/status_im/ui/screens/wallet/send/subs.cljs

```diff
@@ -1,27 +1,76 @@
 (ns status-im.ui.screens.wallet.send.subs
-  (:require [re-frame.core :as re-frame]))
-
-(re-frame/reg-sub :camera-dimensions
-  (fn [db]
-    (get-in db [:wallet/send-transaction :camera-dimensions])))
-
-(re-frame/reg-sub :camera-flashlight
-  (fn [db]
-    (get-in db [:wallet/send-transaction :camera-flashlight])))
-
-(re-frame/reg-sub
-  :wallet.send/sign-enabled?
-  :<- [:get-in [:wallet/send-transaction :amount]]
-  :<- [:get-in [:wallet/send-transaction :to-address]]
-  :<- [:get-in [:wallet/send-transaction :amount-error]]
-  (fn [[amount to-address amount-error]]
-    (and
-      (nil? amount-error)
-      (not (nil? to-address)) (not= to-address "")
-      (not (nil? amount)) (not= amount ""))))
-
-(re-frame/reg-sub
-  :wallet.send/sign-password-enabled?
-  :<- [:get-in [:wallet/send-transaction :password]]
-  (fn [password]
+  (:require [re-frame.core :as re-frame]
+            [status-im.i18n :as i18n]
+            [status-im.utils.money :as money]
+            [status-im.utils.hex :as utils.hex]))
+
+(re-frame/reg-sub ::send-transaction
+  :<- [:wallet]
+  (fn [wallet]
+    (:send-transaction wallet)))
+
+(re-frame/reg-sub :wallet.send/camera-dimensions
```

Comment:
camera stuff could go in a camera key since only once view consumes the three anyway but I kept that for later